docs/cli/argo_archive_list.md

```diff
@@ -6,6 +6,25 @@ list workflows in the archive
 argo archive list [flags]
 ```
 
+### Examples
+
+```
+# Display all archive lists in the default format (wide)
+argo archive list
+
+# Output in JSON format
+argo archive list -o json
+
+# Filter list based on a specific label selector
+argo archive list -l key1=value1,key2=value2
+
+# Fetch the list in chunks instead of all at once
+argo archive list --chunk-size 100
+
+# Display help information for the command
+argo archive list -h
+```
```

Comment:
This is failing CI Codegen as it must match the CLI code verbatim

Reply:
@agilgur5 I don't know why I keep failing the CI test. Can you tell me a little more?

Reply:
@leehosu That error should hopefully be fixed with https://github.com/argoproj/argo-workflows/pull/13660. After that's merged, click the "Update branch" button at the bottom of the PR

Reply:
@MasonM Thank you for your help! I hope it's merge...